scripts/bootstrap.sh

```diff
@@ -44,20 +44,29 @@ fi
 
 upgrade_python=false
 install_git=false
+
 # Python is such a hard dependency of Salt that we have to upgrade it outside of 
 # Salt to avoid changing it while it is running
-python_version=$(dpkg-query -W --showformat='${Version}' python2.7) # e.g. 2.7.5-5ubuntu3
-if dpkg --compare-versions "$python_version" lt 2.7.12; then
-    # we used this, which is not available anymore, to provide a more recent Python 2.7
-    # let's remove it to avoid apt-get update errors
-    rm -f /etc/apt/sources.list.d/fkrull-deadsnakes-python2_7-trusty.list
-    # provides python2.7[.13] package and some dependencies
-    add-apt-repository ppa:jonathonf/python-2.7
-    # provides a recent python-urllib3 (1.13.1-2) because:
-    # libpython2.7-stdlib : Breaks: python-urllib3 (< 1.9.1-3) but 1.7.1-1ubuntu4 is to be installed
-    # due to the previous PPA
-    add-apt-repository ppa:ross-kallisti/python-urllib3
+if ! command python2.7; then
+    # python not found
```

Comment:
I guess because only Python 3 is there... maybe rename the variable to `upgrade_python2.7`